torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -235,45 +270,62 @@ def get_aten_name(
     def get_function_overloads(
         self,
         node: torch.fx.Node,
-        aten_name: str,
+        domain: str,
+        op_name: str,
+        overload: Optional[str],
         diagnostic_context: diagnostics.DiagnosticContext,
-    ) -> Set[Union["onnxscript.OnnxFunction", "onnxscript.TracedOnnxFunction"]]:
+    ) -> Tuple[
+        Set[registration.SymbolicFunction], Optional[Set[registration.SymbolicFunction]]
+    ]:
         """Get the function overloads from the registry.
 
         Args:
             node: The node to get the function overloads for.
-            aten_name: The aten name of the node.
+            domain: The domain of the node.
+            op_name: The name of the node.
+            overload: The FX overload of the node. It's None if the node is default overload.
             diagnostic_context: The diagnostic context to use for reporting errors.
 
         Returns:
             Set of function overloads.
         """
+
         function_group = None
+        custom_function_group = None
```

Comment:
I would comment on the "why" on having two groups here for future readers

Reply:
Good to also annotate the type since it is unclear from the context

Reply:
Tried to put on a NOTE. LMK if it's good enough.